Added release blurb and fixed grammar in api breaking sentence

Qchristensen opened the pull request:
<!-- Please read and fill in the following template, for an explanation of the sections see:
https://github.com/nvaccess/nvda/wiki/Github-pull-request-template-explanation-and-examples
Please also note that the NVDA project has a Citizen and Contributor Code of Conduct which can be found at https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md. NV Access expects that all contributors and other community members read and abide by the rules set out in this document while participating or contributing to this project. This includes creating or commenting on issues and pull requests. 

Please initially open PRs as a draft. See https://github.com/nvaccess/nvda/wiki/Contributing
-->

### Link to issue number:

### Summary of the issue:
Needed a release blurb for 2021.1 and also there was a grammar error in the "API breaking sentence" in the what's new document.

### Description of how this pull request fixes the issue:

With words

### Testing strategy:

Read text, and ran past others when writing.

### Known issues with pull request:

### Change log entries:

For Developers

### Code Review Checklist:

<!--
This checklist is a reminder of things commonly forgotten in a new PR.
Authors, please do a self-review and confirm you have considered the following items.
Mark items you have considered by checking them.
You can do this when editing the Pull request description with an x: `[ ]` becomes `[x]`.
You can also check the checkboxes after the PR is created.
-->

- [X] Pull Request description is up to date.
- [ ] Unit tests.
- [ ] System (end to end) tests.
- [ ] Manual tests.
- [ ] User Documentation.
- [X] Change log entry.
- [ ] Context sensitive help for GUI changes.

Qchristensen committed:
Added release blurb and fixed grammer in api breaking sentence

josephsl commented:
Hi,

In the release blurb: I think there is still a grammatical error with "add-on API" sentence, and may I suggest splitting that note into a separate line/paragraph?

Thanks.

feerrenrut commented on user_docs/en/changes.t2t:
```diff
@@ -4,6 +4,8 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.1 =
+This release includes optional experimental support for UIA in Excel and
+Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and maths symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This releas ...
```
We normally put sentences on new lines to make life easier for translators. Also this copied the grammar error before the fix.
```suggestion
This release includes optional experimental support for UIA in Excel and Chromium browsers.
There are fixes for several languages, and for accessing links in Braille.
There are updates to Unicode CLDR and maths symbols, eSpeak-NG and LibLouis.
Many bug fixes and improvements, including in Office, Visual Studio and several languages.
Note: this is an Add-on API compatibility breaking release.
Add-ons will need to be re-tested and have their manifest updated.
This release also drops support for Adobe Flash.

```

codeofdusk commented on user_docs/en/changes.t2t:
```diff
@@ -4,6 +4,8 @@ What's New in NVDA
 
 = 2021.1 =
+This release includes optional experimental support for UIA in Excel and
+Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and maths symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is a Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This releas ...
```
Use "mathematics" in international contexts as the abbreviation [differs by English variety](https://en.wiktionary.org/wiki/math#Noun_2). Also fix a/an.

```suggestion
Chromium browsers. There are fixes for several languages, and for accessing links in Braille. There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis. Many bug fixes and improvements, including in Office, Visual Studio and several languages. Note: this is an Add-on API compatibility breaking release. Add-ons will need to be re-tested and have their manifest updated. This release also drops support for Adobe Flash.

```

Qchristensen replied:
Thanks Bill, I've made that change, and another grammatical error someone pointed out via email.

Qchristensen committed:
Update user_docs/en/changes.t2t

Thanks Reef!

Co-authored-by: Reef Turner <feerrenrut@users.noreply.github.com>

Qchristensen committed:
Updated wording as per user requests

Updated "maths" to "mathematics" as per Bill's request - thanks Bill!

Also updated the wording of one of the bug fixes to fix the grammar.  Changed from "... to use NVDA in a languages ..." to "... to use NVDA in languages ..."
(Thanks Susanne via email for pointing that one out)

codeofdusk commented on user_docs/en/changes.t2t:
```diff
@@ -4,6 +4,14 @@ What's New in NVDA
 = 2021.1 =
+This release includes optional experimental support for UIA in Excel and Chromium browsers.
+There are fixes for several languages, and for accessing links in Braille.
+There are updates to Unicode CLDR and mathematics symbols, eSpeak-NG and LibLouis.
```
```suggestion
There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis.

```

Qchristensen committed:
Update user_docs/en/changes.t2t

Updated "Mathematics" to "Mathematical"

Co-authored-by: Bill Dengler <codeofdusk@gmail.com>

feerrenrut reviewed (approved):
Looks good @Qchristensen
